package.json

```diff
@@ -34,6 +34,6 @@
     "xml": "^1.0.0"
   },
   "peerDependencies": {
-    "mocha": "^2.2.5"
+    "mocha": "^3 || ^2.2.5"
```

Comment:
Specifying the version range this way feels a bit clunky. Do you think `>=2.2.5` reads more intuitive? Otherwise, I have no problem supporting mocha 3x, so long as @clayreimann is good with it.